.github/workflows/sec.yml

```
@@ -9,11 +9,12 @@ on:
       - 'release/*'
 
 jobs:
-  Sec:
-    uses: komune-io/fixers-gradle/.github/workflows/sec-workflow.yml@main
+  sec:
+    uses: ./.github/workflows/sec-workflow.yml
```

actionlint (1.7.10)

could not read reusable workflow file for "./.github/workflows/sec-workflow.yml": open /home/jailuser/git/.github/workflows/sec-workflow.yml: no such file or directory

(workflow-call)